src/Raven.Server/ServerWide/Commands/Indexes/PutIndexCommand.cs

```diff
@@ -50,6 +58,31 @@ public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
             return null;
         }
 
+        private bool InitializeStaggeredDeployment(DatabaseRecord record)
+        {
+            // What happens here though? No members.. do we resend the command? How do we make sure this will run again?
+            if (record.Topology.Members.Count == 0)
```

Comment:
This can happen if all the members are on rehab, but we can still create new indexes in this mode.
I think that in this case, we need to use AllMembers, not Members.

Need to also take into account the fact that we also have rehab and promotable instances as well.

Reply:
What happens if we have just one node?